### PR TITLE
Added Stafi Network logos & endpoints

### PR DESCRIPTION
**Pull request by Tore19**
Description

Added endpoints, logos and spec of Stafi Network

https://stafi.io
https://stafi.subscan.io

### CODE REVIEW

File: packages/apps-config/src/api/spec/index.ts [hunk 1]
```diff
@@ -40,6 +41,7 @@ export default {
   'nodle-chain': nodle,
   plasm,
   robonomics,
+  stafi,
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:44]: Just move this 2 lines down.

File: packages/apps-config/src/ui/general/index.ts [hunk 2]
```diff
@@ -63,6 +64,7 @@ const nodeColors: Record<string, any> = [
   ['centrifuge chain', nodeCentrifuge],
   ['Centrifuge Chain Node', nodeCentrifuge],
   ['edgeware node', nodeEdgeware],
+  ['Stafi node', nodeStafi],
```
Comment (jacogr) [inline packages/apps-config/src/ui/general/index.ts:67]: 2 lines down (alpahbetical)

File: packages/apps-config/src/ui/logos/index.ts [hunk 3]
```diff
@@ -87,6 +88,8 @@ const nodeLogos: Record<string, any> = [
   ['parity-polkadot', nodePolkadot],
   ['Plasm', nodePlasm],
   ['Plasm Node', nodePlasm],
+  ['Stafi Node', nodeStafi],
+  ['Stafi', nodeStafi],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:92]: 2 lines down, alphabetical

File: packages/apps-config/src/ui/logos/index.ts [hunk 4]
```diff
@@ -114,6 +117,7 @@ const namedLogos: Record<string, any> = {
   nodle: nodeNodle,
   phala: nodePhala,
   plasm: nodePlasm,
+  stafi: nodeStafi,
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:120]: A couple of lines down.

**Review by jacogr — COMMENTED** [review]
Thank you very much.

Made some small comments where the liniting also complained. (It is easier maintaining these when it is alphabetical, the number is getting quite large. So, yes, small pain initially, but makes it easier in the longer-run)

**Comment by Tore19** [comment]
Sure, make sense! 😃

**Review by jacogr — CHANGES_REQUESTED** [review]
Alphabetical, as discussed. (Small pain now for ongoing maintenance)

**Comment by Tore19** [comment]
> Alphabetical, as discussed. (Small pain now for ongoing maintenance)

Sorry for late. Done!

**Review by jacogr — APPROVED** [review]
Thank you! Really happy to have this.

**Pull request merged**